compiler/p/codegen/OMRRegisterDependency.cpp

```diff
@@ -233,7 +233,7 @@ OMR::Power::RegisterDependencyConditions::RegisterDependencyConditions(
 
       if (reg!=NULL /* && reg->getKind()==TR_GPR */)
 	 {
-	 if (child->getHighGlobalRegisterNumber() > -1)
+	 if (TR::Compiler->target.is32Bit() && child->getType().isInt64())
```

Comment:
Yes, this would probably be equivalent. I made these change to supplement what Julian fixed for the same issue in other P codegen parts, where he used the same isInt64 check. For consistency reason, I think we can use the same check as in other places.